src/components/Menu-Badge/MenuBadge.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import { MenuBadge } from './MenuBadge';
```

Comment:
Story imports the component directly which is fine, but this file should use Storybook CSF typing to get type-safety and proper controls (Meta/StoryObj). Right now the default export is untyped.

**Sugerencia:**
```typescript
Use @storybook/react types and typed Meta/StoryObj. Example:

import type { Meta, StoryObj } from '@storybook/react';

const meta: Meta<typeof MenuBadge> = {
  title: 'Components/MenuBadge',
  component: MenuBadge,
};

export default meta;

export const Default: StoryObj<typeof MenuBadge> = { args: { label: 'Dashboard', variant: 'default' } };

```